Dockerfile.compute-node

```diff
@@ -654,14 +650,42 @@ RUN case "${PG_VERSION}" in \
 
 USER root
 
+#########################################################################################
+#
+# Layer "rust extensions"
+# This layer is used to build `pgrx` deps
+#
+#########################################################################################
+FROM build-deps AS rust-extensions-build-pgrx
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
+
+RUN apt-get update && \
+    apt-get install -y curl libclang-dev cmake && \
+    useradd -ms /bin/bash nonroot -b /home
+
+ENV HOME=/home/nonroot
```

Comment:
can we create additional layer with installed rust and then two branches with pgrx and pgx?

Reply:
Honestly, I wouldn't bother, we have only `pg_graphql` and `pg_jsonschema` left, after their releases we can drop the old layer completely.

Reply:
ok